data/ideas.json

```
@@ -34,5 +34,11 @@
         "description": "This repository is about the code to extract Convolutional-neural-network layers and visualise the filters and featuremaps",
         "level": "basic",
         "tags": "feature-maps-and-filter-visualizer, machine-learning"
+    },
+    {
+        "title": "Rock, Paper, Scissors",
+        "description": "Create a rock paper scissors game! Try recreating the classic game, or put your own fun twist on it.",
+        "level": "beginner",
```

The level only support basic, intermediate and advance
```suggestion
        "level": "basic",
```